framework/config/config.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, root_dir, owtf_pid):
         # Available profiles = g -> General configuration, n -> Network plugin
         # order, w -> Web plugin order, r -> Resources file
         self.initialize_attributes()
-        self.LoadFrameworkConfigFromFile(os.path.join(self.RootDir, 'framework', 'config', 'framework_config.cfg'))
+        self.LoadFrameworkConfigFromFile(self.select_config_path(os.path.join(os.path.expanduser('~'), '.owtf', 'configuration', 'framework_config.cfg')))
```

Comment:
Shouldn't
``self.select_config_path(os.path.join(os.path.expanduser('~'), '.owtf', 'configuration', 'framework_config.cfg')``
be in fact
``self.select_config_path('framework_config.cfg')``
and the `os.path.thingy` moved to `self.select_config_path()`?